pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -327,3 +327,12 @@ func ValidateNodeResourcesFitArgs(path *field.Path, args *config.NodeResourcesFi
 	}
 	return allErrs.ToAggregate()
 }
+
+// ValidateDynamicResourcesArgs validates that DynamicResourcesArgs are correct.
+func ValidateDynamicResourcesArgs(path *field.Path, args *config.DynamicResourcesArgs) error {
+	var allErrs field.ErrorList
+	if args.FilterTimeout != nil && args.FilterTimeout.Duration < 0 {
```

Comment:
Oh, could `Duration` be nil actually? (like people are setting -10s on `FilterTimeout` in that case?)